src/runtime-rs/crates/service/src/manager.rs

```diff
@@ -72,8 +152,13 @@ impl ServiceManager {
         Ok(())
     }
 
-    pub fn cleanup(id: &str) -> Result<()> {
-        RuntimeHandlerManager::cleanup(id)
+    pub fn cleanup(sid: &str) -> Result<()> {
+        let temp_dir = [KATA_PATH, sid].join("/");
+        if std::fs::metadata(temp_dir.as_str()).is_ok() {
+            info!(sl!(), "cleanup the sandbox temp_dir: {:?}", &temp_dir);
+            fs::remove_dir_all(temp_dir).ok();
```

Comment:
What's `.ok()` used for?

Reply:
I think it is equal to `let _ = fs::remove_dir_all(temp_dir);` , the purpose is to ignore result.

Reply:
Thanks for your review, the purpose is to ignore the result, and add a comment.